src/base-element.js

```diff
@@ -82,9 +84,18 @@ import {vsyncFor} from './vsync';
  * before buildCallback and it might be called multiple times including
  * after layoutCallback.
  *
- * The documentInactiveCallback is called when the document becomes
- * inactive. E.g. when the user swipes away from the document or
- * focuses a different tab.
+ * The pauseCallback is called when when the document becomes inactive, e.g.
+ * when the user swipes away from the document. In these situations, any
```

Comment:
Is that not https://github.com/ampproject/amphtml/pull/1637/files#diff-1c49a801f8087eb75dd39787699dfd57R62?